botcore/verify.py

```diff
@@ -146,11 +151,14 @@ def get_code(self, user):
         elif ans == "n":
             await self.verify_proc_request_email(member)
         else:
-            await member.send("Please type 'y' or 'n'.")
+            await member.send("Please type `y` or `n`.")
 
     async def verify_proc_request_zid(self, member):
         await member.send("What is your 7 digit student number, not including the 'z' at the start?")
-        self.verifying[member.id].state = self.STATE_AWAIT_ZID
+
+        next_state = self.STATE_AWAIT_ZID
+        self.db.collection("members").document(str(member.id)).update({"_state": next_state})
```

Comment:
i see this line repeated a lot. This is where abstracting db to another class comes in handy :)

Reply:
Abstracting to db has been done!